pytorch/training/docker/1.11/py3/Dockerfile.cpu

```diff
@@ -259,12 +259,19 @@ RUN conda install -y -c conda-forge \
 
 RUN pip install --upgrade pip --trusted-host pypi.org --trusted-host files.pythonhosted.org \
  && pip install --no-cache-dir -U \
-    smdebug==${SMDEBUG_VERSION} \
+   #  smdebug==${SMDEBUG_VERSION} \
```

Comment:
can we add more specific comment about adding back smdebug?

Reply:
updated